dali/pipeline/operator/argument.cc

```diff
@@ -41,7 +41,8 @@ inline std::shared_ptr<Argument> DeserializeProtobufVectorImpl(const DaliProtoPr
   auto args = arg.extra_args();
   std::vector<T> ret_val;
   for (auto& a : args) {
-    const T& elem = DeserializeProtobuf(a)->Get<T>();
```

Comment:
After changing `Argument::Get` to returning a reference, this became a bug: returning a reference to a field of the result of `DeserializeProtobuf`.